addons/web_editor/controllers/main.py

```diff
@@ -182,6 +183,12 @@ def _update_checklist_recursive (self, li, checked, children=False, ancestors=Fa
 
         return True
 
+    @http.route('/web_editor/attachment/fetch', type='json', auth='user', methods=['POST'], website=True)
+    def fetch_attachments(self, domain=None, fields=None, offset=0, limit=None, order=None, **read_kwargs):
```

Comment:
the 3 routes below seems ordered alphabetically, so it would be better to keep order